dev-client/src/components/RadioBlock.tsx

```diff
@@ -41,14 +41,15 @@ export const RadioBlock = <T extends string>({
   label,
   options,
   allDisabled = false,
-  groupProps: {onChange, ...radioGroupProps},
+  groupProps: {onChange, value: groupValue, ...radioGroupProps},
   labelProps,
 }: Props<T>) => {
   return (
     <FormControl>
       <FormControl.Label {...labelProps}>{label}</FormControl.Label>
       <Radio.Group
         onChange={onChange as (_: string) => void}
+        value={groupValue ?? ''}
```

Comment:
question: if you don't use `??`, what does groupValue default to?

Reply:
the NativeBase Radio.Group component uses `''` instead of `undefined` as the value to represent "no radio button selected", so this accommodates that quirk (we didn't have any radio groups that could have no options selected until this PR)